src/modules/github/github.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getReposActionApi } from "./github.actions";
+
+const Github = () => {
+  const init = { username: "" };
+  const [state, setState] = useState(init);
+  const [searchTerm, setSearchTerm] = useState("");
+  const handleChange = (e) => {
+    setState({
+      ...state,
+      [e.target.name]: e.target.value,
+    });
+    console.log(state);
+  };
+  const repos = useSelector((props) => props.repos);
+  const error = useSelector((props) => props.error);
```

Comment:
Musisz pamiętać, że jak "łączysz" magazyny to one występują pod inną nazwą teraz więc musisz dodać: `props.github.repos`

PS. Nie używałbym nazwy parametru `props` bo może ona mylić. Używamy jej w komponentach, a nie w danych z magazynu :)